cypress/integration/tests/developer/applicationinventory/assessment/filter.test.ts

```diff
@@ -251,4 +252,31 @@ describe("Application inventory filter validations", { tags: "@tier2" }, functio
         exists(application1.name);
         clickByText(button, clearAllFilters);
     });
+
+    it("Artifact type filter validations", function () {
+        // For application must have Binary group,artifact and version
+        const application2 = new Application(
```

Comment:
const name is unique in this method so it can be just application instead of application2.